e2e/portalicious/tests/ManageTeam/AllSystemUsersCanBeAddedToProjectTeam.spec.ts

```diff
@@ -16,7 +16,8 @@ const expectedAssignedUsers = ['admin@example.org'];
 const expectedAvailablesystemUsers = [
   'program-admin@example.org',
   'view-user@example.org',
-  'kobo-user@example.org',
+  'kobo+registration_country@121.global',
+  'kobo+validation_country@121.global',
```

Comment:
```suggestion
  'kobo+registration_country@example.org',
  'kobo+validation_country@example.org',
```

We should not use real-world specific email-addresses in our (publicly shared) code. 

That's why we have the ENV-variables. So that we can set actual values per-instance.